### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by lewisnyman** [matt-curtis/Fluid-for-Sketch#47]
Hi! Thanks so much for making this plugin. It's solving big problem I've had with all drag-n-drop UI tools, not just Sketch.

I really want to make suggestions work help make this better, but I don't know how to start without stepping on toes. I guess the first thing I can help with is establishing contribution guidelines?

See: https://github.com/blog/1184-contributing-guidelines

**Comment by HBehrens** [comment]
Yes, I was also wondering: "How do you rebuild `MCSketchConstraints.bundle`?" – do you plan to release its source and share your build process for it?

**Comment by studiostoll** [comment]
@HBehrens +1

**Comment by matt-curtis** [comment]
Contribution guidelines are in the works. For now though, as far as suggestions/features go feel free to open an issue (so long as it isn't a duplicate, search first ofc).

Where the source is concerned, I'm planning on releasing it alongside the 1.8 (ish?). There are several places in the current code that I want to tidy up, as well as portions that are being rewritten entirely, before I do.

**Comment by studiostoll** [comment]
awesome, @matt-curtis :-)
